dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/data_config_panel_item.tsx

```diff
@@ -94,6 +95,10 @@ export const DataConfigPanelItem = ({ fieldOptionList, visualizations }: any) =>
       setConfigList({
         dimensions: [{ bucketSize: '', bucketOffset: '' }],
       });
+    } else if (visualizations.vis.name === visChartTypes.Metrics) {
+      setConfigList({
+        metrics: [initialConfigEntry],
+      });
```

Comment:
please take a pull of latest code, we are using switch case here in place of if-else